docs/software/node.md

```diff
@@ -77,12 +79,20 @@ To start a multi-node IRISnet network locally, you need to follow the steps belo
 
 * Prepare home directory: create a unique home directory for each node
 * Initialization: according to the above `Initialize node` steps, initialize each node in the respective home directory (note that you need to use the same chain-id)
-* Configure genesis: select the home directory of one of the nodes to configure the genesis, refer to the `Config genesis` steps, configure the account address and initial balance of each node, and copy the files from each node {path_to_your_home}/config/gentx to the current one. Execute `iris collect-gentxs` to generate the final genesis.json. Finally, copy the genesis.json to the home directory of each node, overwriting the original genesis.json.
-* Configure config.toml: modify {path_to_your_home}/config/config.toml in the home directory of each node, assign different ports to each node, query the node-id of each node through `iris tendermint show-node-id`, and then add the `node-id@ip:port` of other nodes in `persistent_peers` so that nodes can connect to each other.
+* Configure genesis: select the home directory of one of the nodes to configure the genesis, refer to the `Config genesis` steps, configure the account address and initial balance of each node, and copy the files from each node <path_to_your_home>/config/gentx to the current one. Execute `iris collect-gentxs` to generate the final genesis.json. Finally, copy the genesis.json to the home directory of each node, overwriting the original genesis.json.
+* Configure config.toml: modify <path_to_your_home>/config/config.toml in the home directory of each node, assign different ports to each node, query the node-id of each node through `iris tendermint show-node-id`, and then add the `node-id@ip:port` of other nodes in `persistent_peers` so that nodes can connect to each other.
 
-In order to simplify the above configuration process, you can use the following commands to automate the initialization of local multi-node and the configuration of genesis and config.toml:
+In order to simplify the above configuration process, you can use the following command to automate the initialization of local multi-node and the configuration of genesis and config.toml:
 ```bash
-iris testnet --v 4 --output-dir ./output --chain-id irishub-test --starting-ip-address 127.0.0.1
+iris testnet --v=2 --output-dir=<path_to_your_home> --chain-id=<chain-id> --starting-ip-address 127.0.0.1
+```
+
+config the port of node1：modify config.toml of node1 under folder <path_to_your_home>/node0/iris/config/，allocate different port for it 
+Start 2 nodes separately:
+
+```bash
+iris start --home=<path_to_your_home>/node0/iris
+iris start --home=<path_to_your_home>/node1/iris
 ```
 
 To start the node to join the public IRIS Testnet, please refer to [Full-Node](../get-started/Full-Node.md)
```

Comment:
To start the node to join the public IRIS Testnet --> To run a full node on the network

Reply:
fixed